### PR TITLE
SISRP-23303 Derive CS ID, student ID from CS Affiliations feed

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.berkeley.edu/browse/SISRP-23303

The cheese stands alone.

### CODE REVIEW

File: spec/models/hub_edos/user_attributes_spec.rb [hunk 1]
```diff
@@ -28,6 +31,50 @@
...
+    end
+    context 'student' do
+      let(:affiliations) do
+        [
+          {type: {code: 'STUDENT'}, status: {code: 'ACT'}},
+          {type: {code: 'UNDERGRAD'}, status: {code: 'ACT'}}
+        ]
+      end
+      it 'does set student_id' do
+        expect(subject[:student_id]).to  eq campus_solutions_id
```
Comment (paulkerschen) [inline spec/models/hub_edos/user_attributes_spec.rb]: Stray space, @raydavis

Comment (raydavis) [inline spec/models/hub_edos/user_attributes_spec.rb]: Sharp eyes, @paulkerschen

**Comment by raydavis** [comment]
@davidlimcheng If you have time, could you check this? Thanks!

**Comment by paulkerschen** [comment]
FWIW looks good to me @raydavis.

**Comment by raydavis** [comment]
IW a lot, @paulkerschen , thank you!

**Comment by davidlimcheng** [comment]
LGTM as well, @raydavis.

**Pull request merged**